<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHighlightsFrame</class>
 <widget class="QFrame" name="CHighlightsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>90</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="filterEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnColor">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="filterList">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>CHighlightsFrame</receiver>
   <slot>addHighlighting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>CHighlightsFrame</receiver>
   <slot>deleteHighlighting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>CHighlightsFrame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CHighlightsFrame</receiver>
   <slot>onTextEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>btnAdd</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>CHighlightsFrame</receiver>
   <slot>onSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnColor</sender>
   <signal>clicked()</signal>
   <receiver>CHighlightsFrame</receiver>
   <slot>changeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addHighlighting()</slot>
  <slot>deleteHighlighting()</slot>
  <slot>onTextEdited(QString)</slot>
  <slot>onSelectionChanged()</slot>
  <slot>changeColor()</slot>
 </slots>
</ui>
